stats.go

```diff
@@ -335,44 +331,37 @@ func (e *statsExporter) viewToMetricDescriptor(ctx context.Context, v *view.View
 // An error will be returned if there is already a metric descriptor created with the same name
 // but it has a different aggregation or keys.
 func (e *statsExporter) createMeasure(ctx context.Context, v *view.View) error {
```

Comment:
nit: May be createMeasure should be renamed to createMetricDescriptorFromView like others

Reply:
Done.